R/count_watershed_data.R

```diff
@@ -259,71 +266,88 @@ count_watershed_data <- function(data_dir, watersheds = NULL){
 
           #---------------------------------------------------------
           # TELECONNTECTION - Number of climate zones watershed crosses
-          get_zonal_data(us_climate, watersheds_select) -> climate_ids
+          get_zonal_data(us_climate, watersheds_select) %>%
+            filter(Group.1 != 128) %>% .[["Group.1"]] -> climate_ids
+
+          return(
+            list(
+              counts = tribble(
+                ~item,                           ~count,
+                "other cities using watershed",  n_other_cities,
+                "hydro plants",                  n_hydro_plants,
+                "thermal plants",                n_thermal_plants,
+               #"reservoirs",                    n_reservoirs,
+                "transfers in",                  n_transfers_into,
+                "transfers out",                 n_transfers_out,
+                "transfers within",              n_transfers_within
+              ),
+              metrics = tribble(
+                ~metric,                           ~unit,   ~value,
+                "watershed area",                  "sq_km",  polygon_area,
+                "total hydro generation",          "MWh",    hydro_gen_MWh,
+                "total thermal generation",        "MWh",    thermal_gen_MWh,
+                "total thermal water consumption", "BCM/yr", thermal_consum_BCM,
+                "total thermal water withdrawals", "BCM/yr", thermal_withdr_BCM
+              # "total reservoir storage",         "BCM",    xxx
+                ),
+              land = land_table,
+              economic_sectors = nlud_table,
+              #irrigation = ????
+              climate_zones = climate_ids
+              )
+            )
+        }
 
-          climate_ids[!(climate_ids$Group.1=="128"),] %>% nrow() -> n_climate_zones
```

Comment:
Just double checking this. The group ID "128" is considered "background" and so this line removed that from being counted as a climate zone. Did you have another plan other than just counting the climate zones?